src/cljs/rems/administration/create_form.cljs

```diff
@@ -134,13 +134,23 @@
   (into {} (for [language languages]
              [language (trim-when-string (get lstr language ""))])))
 
+(defn- string-is-not-empty [s]
+  (if (string? s)
+    (not (str/blank? s))
+    false
+    ))
+
 (defn- build-request-field [field languages]
   (merge {:field/id (:field/id field)
           :field/title (build-localized-string (:field/title field) languages)
           :field/type (:field/type field)
           :field/optional (if (common-form/supports-optional? field)
                             (boolean (:field/optional field))
                             false)}
+         (when (some true? 
+                (mapv (fn [l] (string-is-not-empty (l (:field/info-text field)))) 
+                   languages))
```

Comment:
this can be simplified to:

```
(some (fn [l] (string-is-not-empty (l (:field/info-text field))))
      languages)
```

Reply:
This should probably also rather be `not-any?` `str/blank?`?